db/sltdbt.c

```diff
@@ -169,7 +176,16 @@ static int handle_op_block(struct ireq *iq)
         rc = ERR_NOT_DURABLE;
     }
 
-    if (rc == RC_INTERNAL_RETRY) {
+    if (rc == 0) {
+        /* Calculate average runtime of last 16 toblock calls, and cap it
```

Comment:
Also is the comment `average runtime of last 16 toblock calls` correct? It seems this is a more like an exponential moving average?